ouroboros-consensus-cardano/tools/db-analyser/Main.hs

```diff
@@ -124,6 +126,16 @@ storeLedgerParser = (StoreLedgerStateAt . SlotNo . read) <$> strOption
   <> metavar "SLOT NUMBER"
   <> help "Store ledger state at specific slot number" )
 
+parseLimit :: Parser Limit
+parseLimit = asum [
```

Comment:
I was expecting to see https://hackage.haskell.org/package/optparse-applicative-0.16.1.0/docs/Options-Applicative.html#v:value (which would require `option` instead of `strOPtion`, I think) used here to specify defaults. But I see that what you've written here at least matches the rest of the option definitions in this module.

I don't recall if there's a difference here. I'm guessing `value` _might_ lead to the default value being displayed in the `--help` message?

Reply:
we are going to revisit the whole interface before we deliver the tool to ledger team and benchmarking team https://input-output.atlassian.net/browse/CAD-3477